src/main/java/de/komoot/photon/App.java

```diff
@@ -30,6 +32,16 @@ public static void main(String[] rawArgs) {
 			jCommander.usage();
 			return;
 		}
+                
+                if(args.getLanguages().equals("all")) {
+                    java.util.ArrayList<String> languageList = new java.util.ArrayList<String>();
+                    for (LanguageCode lang: LanguageCode.values())
+                    {
+                        languageList.add(lang.name());
+                    }
+                    String[] languages = languageList.toArray(new String[languageList.size()]);
+                    args.setLanguages(String.join(",", languages));
+		}
```

Comment:
allowing all languages may have unwanted side effects: every language needs some extra storage, having hundreds of languages may multiply the index by 10 (I haven't done the exact math). this parameter promotes wrong usage, it should be mentioned in the README and a warn message is appropriate: 'log.warn("...")'